### PR TITLE
Add Kontroll.app version 1.0

### PR DESCRIPTION
**Pull request by vinkla**
Kontroll your Mac using your iOS devices. http://kontroll.io/

### CODE REVIEW

File: Casks/kontroll.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  sha256 :no_check
+
+  url 'http://kontroll.io/download/Kontroll.zip'
+  name 'Kontroll'
+  homepage 'http://kontroll.io/'
+  license :gratis
+
+  app 'Kontroll.app'
+
+  zap :delete => [
```
Comment (vitorgalvao) [inline Casks/kontroll.rb]: We don’t need an array, here. It could be `zap :delete => '~/Library/Preferences/com.vinkla.Kontroll.plist'`.

Comment (vinkla) [inline Casks/kontroll.rb]: Done.

**Comment by vinkla** [comment]
Thanks @vitorgalvao!

**Comment by vitorgalvao** [comment]
Thank you, as always.

**Pull request merged**